globalizationStrings.h

```diff
@@ -0,0 +1,58 @@
+#ifndef GLOBALIZATIONSTRINGS_H
+#define GLOBALIZATIONSTRINGS_H
+/*--------------------language used------------------*/
+//we can define the language using parameter -D <lenguageDefinition>. Example: g++ -Wall -Wextra -D GERMAN -o checkerTest *.cpp
+//#define ENGLISH
+//#define GERMAN
+//#define SPANISH
+/*----------------------------------------------------*/
+#include <string>
+#include <map>
+#include "Battery.h"
+typedef std::string globalizationString;
+
+#ifdef ENGLISH
```

Comment:
Nice isolation of strings into a file!
Consider this: We add a new language, but later add a string in English. We forget to translate this into one of the other languages. Testing this is hard, since we have to make a binary in each language and check the strings in all test cases. Is there an easier way to test it?